src/js/constants.js

```diff
@@ -5,6 +5,7 @@ export const MediaUIEvents = {
     MEDIA_UNMUTE_REQUEST: 'mediaunmuterequest',
     MEDIA_VOLUME_REQUEST: 'mediavolumerequest',
     MEDIA_SEEK_REQUEST: 'mediaseekrequest',
+    MEDIA_SELECT_AIRPLAY_REQUEST: 'mediaselectairplayrequest',
```

Comment:
Would it be wrong to just say `MEDIA_AIRPLAY_REQUEST`?

Reply:
Ok cool, makes sense. My S1 is `MEDIA_AIRPLAY_REQUEST`, and `AIRPLAY`, from the stance of the user is attempting to airplay, and doesn't necessarily know there's a select menu coming. And "airplay" itself is a verb if you squint at it. So unless you see something like "there's gonna be an `unselect-airplay`" user action that would make this weird, let's simplify it.